sklearn/model_selection/tests/test_search.py

```diff
@@ -1172,20 +1302,14 @@ def test_grid_search_cv_splits_consistency():
                        cv=KFold(n_splits=n_splits))
     gs2.fit(X, y)
 
-    def _pop_time_keys(cv_results):
-        for key in ('mean_fit_time', 'std_fit_time',
-                    'mean_score_time', 'std_score_time'):
-            cv_results.pop(key)
-        return cv_results
-
     # OneTimeSplitter is a non-re-entrant cv where split can be called only
     # once if ``cv.split`` is called once per param setting in GridSearchCV.fit
     # the 2nd and 3rd parameter will not be evaluated as no train/test indices
     # will be generated for the 2nd and subsequent cv.split calls.
     # This is a check to make sure cv.split is not called once per param
     # setting.
-    np.testing.assert_equal(_pop_time_keys(gs.cv_results_),
-                            _pop_time_keys(gs2.cv_results_))
+    np.testing.assert_equal(_pop_cv_results_time_keys(gs.cv_results_),
```

Comment:
Consider whether `{k: v for k, v in gs.cv_results_ if not k.endswith('_time')}` would make this more readable